deepchecks/nlp/text_data.py

```diff
@@ -733,6 +742,52 @@ def is_sampled(self, n_samples: t.Optional[int]):
             return False
         return self.n_samples > n_samples
 
+    def describe(self, n_properties_to_show: t.Optional[int] = 4, properties_to_show: t.Optional[t.List[str]] = None):
+        """Provide holistic view of the data.
+
+        Generates the following plots:
+        1. Label distribution
+        2. Statistics about the data such as number of samples, annotation ratio, list of metadata columns, list of
+        text properties and so on.
+        3. Property distribution for the text properties defined either by n_properties_to_show or properties_to_show
+        parameter.
+
+        Parameters
+        ----------
+        n_properties_to_show : int, default 4
+            Number of properties to consider for generating property distribution graphs. If properties_to_show
+            is provided, this value is ignored.
+        properties_to_show : List[str], default None
+            List of property names to consider for generating property distribution graphs. If None, all the
+            properties are considered.
+
+        Returns
+        -------
+        Displays the Plotly Figure.
+        """
+        properties = []
+        all_properties_data = pd.DataFrame()
+        if self._properties is None and properties_to_show is not None:
+            raise DeepchecksValueError('No properties exist!')
```

Comment:
can just call the following (will throw an exception if no properties are loaded)
_ = self.properties

Reply:
I think, in this case we only want to raise an exception if the user has specified a list of property names, else even if properties does not exist, we still want to display the label and table figures.